### PR TITLE
Height of github stats has been resized: (read description)

### PR DESCRIPTION
**Pull request by SrNightmare09**
The height of the github stats were previously in separate lines while visiting <a href = https://github.com/DenverCoder1/DenverCoder1>PROFILE REPO</a>. This change doesn't, in any way affect the README stats except for the height. Except for making sure the cards stay in the same line while in the repository view.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,8 +200,8 @@
...
-  <a href="https://github.com/anuraghazra/github-readme-stats"><img alt="DenverCoder1's Top Languages" src="https://denvercoder1-github-readme-stats.vercel.app/api/top-langs/?username=DenverCoder1&langs_count=8&layout=compact&theme=react&hide_border=true&bg_color=1F222E&title_color=F85D7F&icon_color=F8D866" /></a>
+    <a href="https://github.com/anuraghazra/github-readme-stats"><img alt="DenverCoder1's Github Stats" src="https://denvercoder1-github-readme-stats.vercel.app/api?username=DenverCoder1&show_icons=true&count_private=true&theme=react&hide_border=true&bg_color=1F222E&title_color=F85D7F&icon_color=F8D866" height = 192px/></a>
+  <a href="https://github.com/anuraghazra/github-readme-stats"><img alt="DenverCoder1's Top Languages" src="https://denvercoder1-github-readme-stats.vercel.app/api/top-langs/?username=DenverCoder1&langs_count=8&layout=compact&theme=react&hide_border=true&bg_color=1F222E&title_color=F85D7F&icon_color=F8D866" height=192px /></a>
```
Comment (DenverCoder1) [inline README.md:204]: Add quotation marks around values and keep consistent spacing.

```suggestion
    <a href="https://github.com/anuraghazra/github-readme-stats"><img alt="DenverCoder1's Github Stats" src="https://denvercoder1-github-readme-stats.vercel.app/api?username=DenverCoder1&show_icons=true&count_private=true&theme=react&hide_border=true&bg_color=1F222E&title_color=F85D7F&icon_color=F8D866" height="192px"/></a>
  <a href="https://github.com/anuraghazra/github-readme-stats"><img alt="DenverCoder1's Top Languages" src="https://denvercoder1-github-readme-stats.vercel.app/api/top-langs/?username=DenverCoder1&langs_count=8&layout=compact&theme=react&hide_border=true&bg_color=1F222E&title_color=F85D7F&icon_color=F8D866" height="192px"/></a>
```

**Review by DenverCoder1 — COMMENTED** [review]
Thanks for the PR! I do agree that it looks better when they are the same height. :heavy_check_mark:

**Comment by SrNightmare09** [comment]
> Thanks for the PR! I do agree that it looks better when they are the same height. ✔️

You said it, I did it :)

**Review by DenverCoder1 — APPROVED** [review]
Looks good :+1:

**Pull request merged**